### ISSUE TITLE
Sonic Boom: Rise of Lyric [BSS#8P]

### ISSUE THREAD

**Issue by slashiee** [slashiee/cemu_graphic_packs#28]
![image](https://cloud.githubusercontent.com/assets/15317421/22723244/943fba84-ed71-11e6-85af-f845e581c6b5.png)
This is the funniest issue I've made ever

**Comment by slashiee** [comment]
![image](https://cloud.githubusercontent.com/assets/15317421/22764469/bb6636a2-ee1f-11e6-8a22-99140c9a9b43.png)

**Comment by slashiee** [comment]
Game might actually need shader edits?
![image](https://cloud.githubusercontent.com/assets/15317421/24840998/8f89bbba-1d2e-11e7-9321-99a3a6540711.png)

**Comment by slashiee** [comment]
```
// shader 008c9e1d05d77d31
// shader 02f608fd47a3764f
// shader 03bc32ed9186c380
// shader 04275d4bc82b636a
// shader 06e034127e42366a
// shader 06e7ca0dc6d86149
// shader 076722bf16b2be24
// shader 076b88c661583176
// shader 08d564173c757f64
// shader 0b62506f2c8ec531
// shader 0b6faaa4b567e57f
// shader 0b6faaa4b567e57f
// shader 0cb1039ba4a187e6
// shader 0ce6b80f25c5461d
// shader 0d4c6226519f41cb
// shader 0d9638569edb7446
// shader 0f07075e5a88249a
// shader 116ccc5f0051909d
// shader 148fce95f7f7613d
// shader 149180130dda0a2b
// shader 149f74ce1cec7102
// shader 158515e758ce97af
// shader 15a266db2255131b
// shader 1651536431144bc1
// shader 1a1a11b206952d0d
// shader 1a5ab795ec8ff59a
// shader 1ebeae368de0813a
// shader 21885420d49e6984
// shader 22ed061f9b9cf5e4
// shader 260ebea6b664d0d9
// shader 2899f62ff3d34e4a
// shader 2b2bfd475abaff82
// shader 2d110d6c2e4dc672
// shader 2e1429ba68b51040
// shader 2f708d66d8336331
// shader 30ee854e50d4b6b4
// shader 33d97358a50a0849
// shader 34a51e251c075474
// shader 366a1d25d714eb98
// shader 3a053ecc46537349
// shader 3e50ba62c1bbdc0d
// shader 3fed5e04bc882323
// shader 3fff84ffcdac5142
// shader 41fc11afdb852a48
// shader 451fd4e15c221ee3
// shader 491537ac0c28fc0d
// shader 4ae07c71d5a75675
// shader 4df51295ecd3974a
// shader 4f9631bfa7dcca1a
// shader 5504d6bfb7d2fdcc
// shader 561d3ec20f9733e0
// shader 5624a10d063f667e
// shader 582426162ccb5283
// shader 5950ef69c47fc479
// shader 5a644cacc36e4b14
// shader 5a7f3046528a8402
// shader 5ad9c494de71bab6
// shader 5aea5aa4892f23ec
// shader 5d5dff27134b1492
// shader 62834bccba02f7c7
// shader 63ff5fa6fdef92f0
// shader 689c3cf81d41ab08
// shader 69b38b163661fc0f
// shader 69ca2de3d77d66ae
// shader 6b2a667d58d97219
// shader 6c1fceea746eceed
// shader 6c27f1dcd6426696
// shader 6c6ffdf3d18c28d3
// shader 6f1e1487b6a57212
// shader 700d621ec71c592f
// shader 7104a61f26db9a6d
// shader 74b97cc84f99a54a
// shader 74f7370037443288
// shader 754b27ee5d4507e6
// shader 75b9a411cf09b71b
// shader 75f385ee28216ace
// shader 77bec4db95dd273c
// shader 780091ea1474a86b
// shader 7873f8352506e476
// shader 78df22935bab8460
// shader 7905a34da78e6e72
// shader 795c00d979046630
// shader 7961670170a0bd45
// shader 797c52e1e9566ac9
// shader 7ab961e566beafbf
// shader 7d1a7e50b5cde501
// shader 7ee624d1d07a8232
// shader 7f65924438afac76
// shader 811039c837b23f85
// shader 8398b24d074b4487
// shader 8546df08c0c2d475
// shader 8688ba4c0dd53866
// shader 87a6b7a951919a1c
// shader 8a3dfc87d0137cab
// shader 8a7a2e43d911074c
// shader 8b4a01a2f9bfa0f2
// shader 8d2204730d5e9a2a
// shader 8d375b2b329182ca
// shader 8da28589693622b4
// shader 9041a5d3568427b6
// shader 91e8cc65eee44fc0
// shader 91e8cc65eee44fc0
// shader 99c06466e13feba6
// shader 9c74b68e18fa9a98
// shader 9d4f8548b67e0834
// shader a3fd57b8d7219211
// shader a6592f1868e08a9c
// shader a6dbe1bced438cca
// shader a7e99799ff0faf0d
// shader a975e05bc2dc54ae
// shader a9d1a40f8beea381
// shader aa274d6ec6172089
// shader ad02c0434de6850f
// shader ae8ed163ebd8fb4c
// shader ae9f3c8c11744c6b
// shader b1764060a7a6873e
// shader b29ab8651bc62417
// shader b3d22fc75ec1995b
// shader b3d8a90bea02e3eb
// shader b46c4d065fac8e43
// shader b4d7776496512e2d
// shader b4da25c02aef9bbe
// shader b6ae347d8eabcb14
// shader b6db43b24ec8dfae
// shader b6dd1a2961872c34
// shader b77d72f983c6a93b
// shader b8992ae1c2f5c3f2
// shader b9b0a926086f660d
// shader ba4b23f6b2a2f2fc
// shader c0c05629fd9b6853
// shader c101241d3a04f09b
// shader ca30a174fc8dd1e2
// shader cda008594322df5d
// shader cdcf2d0990a9b363
// shader d020678c57b9f3aa
// shader d27c50bb65261423
// shader d42295dc01909eb5
// shader d507c82a0a592e51
// shader d55fff1505208e7d
// shader d6b0f2a7d940361a
// shader d7cd5d72fc9dca7f
// shader da89695e43e37cc8
// shader dd66a55ee3cf41e8
// shader dda7fae342abfee5
// shader de8419b7de4aa0f9
// shader dfd9101aea34134b
// shader e074b35ebbffd472
// shader e12c05428b637085
// shader e33e76a07c709703
// shader e5f1d94fb2020c84
// shader e807aa7572a5b203
// shader e8e02bac6d8c46fe
// shader eb9be3599c8d1789
// shader ebc880d1636f8173
// shader ebe2b81c296db8b4
// shader ed04c4954aea1d56
// shader ed343b81ee535656
// shader ee652b036ceee876
// shader eec5a82161620328
// shader efc0425dc37caeda
// shader f207e57c4e0685c4
// shader f31316bd47e7ad0c
// shader f6a044f6819e8324
// shader f888b8792fbd79bf
// shader f8b8fbe250d89c78
// shader fb5bfed588ae1178
// shader fcdf428d07c38fe7
// shader fe7af6607522b9f2

this'll be fun.

**Comment by slashiee** [comment]
Too lazy to find the shader, excluding format 0x019 fixed it

**Comment by slashiee** [comment]
Fixed with 92049f7

**Comment by slashiee** [comment]
0x019 breaks shadows, fuck this game
![image](https://cloud.githubusercontent.com/assets/15317421/24879933/48ee3f7c-1ded-11e7-8b3a-2ecfd5e1996f.png)

**Comment by LLCooLM495** [comment]
Why would you ever want this game to run? :P

**Comment by slashiee** [comment]
@LLCooLM495 Because I hate myself

**Comment by slashiee** [comment]
This might be fixed with 1.7.6's proper gl_FragCoord scaling. I will check to make sure that this is the case. If it is fixed, I will close this issue when 1.7.6 releases publicly.

**Comment by slashiee** [comment]
![image](https://cloud.githubusercontent.com/assets/15317421/26388088/286adb3e-4007-11e7-9161-0737adbdb065.png)
